src/vulkan-renderer/wrapper/glfw_context.cpp

```diff
@@ -0,0 +1,21 @@
+#include "inexor/vulkan-renderer/wrapper/glfw_context.hpp"
+
+namespace inexor::vulkan_renderer::wrapper {
+
+GLFWContext::GLFWContext(GLFWContext &&other) noexcept {}
```

Comment:
Since the move constructor doesn't do anything, you can just do `= default;` in the header file.

Reply:
Done.